skrf/tests/tmp_skrf_oneport_powerwave.s1p

```diff
@@ -8,8 +8,8 @@
 ! YY = 1
 !Data is not renormalized
 !
-!Created with skrf (http://scikit-rf.org).
-!Data is not renormalized
+! Created with skrf (http://scikit-rf.org).
+! Data is not renormalized according to the power convention.
```

Comment:
I find this sentence confusing. How about splitting into two sentences like:

```
! Data is not renormalized
! S-parameter uses the "{}" definition 
```
?

Reply:
Changes made as suggested.